### DISCUSSION TITLE
Problem with generation reports: Gross and invalid traffic

### DISCUSSION THREAD

**Discussion by mainefol** [amzn/ads-advanced-tools-docs#255] [Q&A]
Why all reports are generated normally, but for gross and invalid we need a longer delay between requests and often get 429 even with a 15 or 20 minute (sometimes even with 24-hour) break? Each time we get "429 throttled". How to fix it?

**Comment by xuqiang97** [comment]
we also have the same problem

**Comment by elizafarley** [comment]
If you haven't been able to resolve this, I'd recommend reaching out to support as it may be a technical bug.
